src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractKeyspaceIntegrationTestBase.java

```diff
@@ -123,7 +123,7 @@ protected void createCollection(String keyspace, String collectionToCreate) {
                   """
                 .formatted(collectionToCreate))
         .when()
-        .post(KeyspaceResource.BASE_PATH, keyspaceName)
```

Comment:
Noticed a minor bug in this test method -- used default `keyspaceName`, not one passed as argument.